### PR TITLE
chore: update desc of activate_api_identities

### PR DESCRIPTION
**Pull request by umairidris**

### CODE REVIEW

**Comment by umairidris** [comment]
@bharathkkb something a bit odd... when i run `sudo make generate_docs` it doesnt seem to update the readmes. Any idea why?

**Comment by bharathkkb** [comment]
@umairidris could be https://github.com/terraform-docs/terraform-docs/issues/105, looks like our image has 0.6.0, so we may need to bump it.

**Comment by umairidris** [comment]
Good catch. Sent https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/pull/814

I think we will need to wait for 0.13 migration? The makefile is pinned to tag 0.12

**Comment by morgante** [comment]
Note that it could also be related to possible issues in our wrapper script: https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/master/infra/build/developer-tools/build/scripts/task_wrapper_scripts/terraform_docs.sh

**Comment by bharathkkb** [comment]
@umairidris after debugging through the GKE module CI failures I think we should also add the `active_api_identities` to [this example ](https://github.com/terraform-google-modules/terraform-google-project-factory/blob/master/examples/gke_shared_vpc) to showcase how to restore `roles/container.serviceAgent`.  We can add a line or two to the [FAQ](https://github.com/terraform-google-modules/terraform-google-project-factory/blob/master/docs/FAQ.md#how-does-the-project-factory-work-with-a-shared-vpc) part explaining the behavior and link to this example. This can also be something we do within our `shared_vpc_access` submodule for GKE and a select few to be transparent to the user in the future or pending a datasource automate it completely.

File: modules/core_project_factory/variables.tf [hunk 1]
```diff
@@ -90,7 +90,12 @@ variable "activate_apis" {
 }
 
 variable "activate_api_identities" {
-  description = "The list of service identities (Google Managed service account for the API) to force-create for the project (e.g. in order to grant additional roles). APIs in this list will automatically be appended to `activate_apis`. Not including the API in this list will follow the default behaviour for identity creation (which is usually when the first resource using the API is created)."
+  description = <<<EOF
```
Comment (bharathkkb) [inline modules/core_project_factory/variables.tf:93]: @umairidris I actually just noticed shouldn't this be 
```suggestion
  description = <<EOF
```

Comment (umairidris) [reply modules/core_project_factory/variables.tf]: D'oh! Good catch. It still isn't passing the docs generation though :(

**Comment by morgante** [comment]
@bharathkkb @umairidris Is this PR ready for merge?

**Comment by umairidris** [comment]
I have regenerated docs with the latest image. PTAL

**Review by morgante — APPROVED** [review]

**Review by bharathkkb — CHANGES_REQUESTED** [review]
@umairidris we need to update here too for Cloud Build
https://github.com/terraform-google-modules/terraform-google-project-factory/blob/2d7fcb474c4492b2fd785e1631352dfcc7ead597/build/lint.cloudbuild.yaml#L24

**Comment by umairidris** [comment]
> @umairidris we need to update here too for Cloud Build
> https://github.com/terraform-google-modules/terraform-google-project-factory/blob/2d7fcb474c4492b2fd785e1631352dfcc7ead597/build/lint.cloudbuild.yaml#L24

Done.

**Review by bharathkkb — APPROVED** [review]

**Pull request merged**